Makefile

```diff
@@ -211,6 +211,7 @@ install: manifests kustomize install-authorino ## Install CRDs into the K8s clus
 
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	kubectl delete -f $(OPERATOR_MANIFESTS)
+	$(MAKE) uninstall-authorino
```

Comment:
`make install` does not install Authorino CRDs. I wonder if `make uninstall` should indeed try to undo what its counterpart did not do.

Reply:
The `make install` does install the Authorino CRD, it is part of `OPERATOR_MANIFESTS`. The adding of the `make uninstall-authorino` is to mirror how the `make install` calls `install-authorino`. This change aligns closer with what its counterpart does.

Reply:
My confusion. [`OPERATOR_MANIFESTS`](https://github.com/Kuadrant/authorino-operator/blob/a8652389ae4de98ca580198594f5a3198080b6de/Makefile#L53) points to [this](https://github.com/Kuadrant/authorino-operator/blob/a8652389ae4de98ca580198594f5a3198080b6de/config/install/manifests.yaml) file, which does _not_ include Authorino CRDs. However, `install` _depends on_ `install-authorino`. Makes sense.

Reply:
I feel like I am missing something. The very first file in that manifest is the [authorino CRD](https://github.com/Kuadrant/authorino-operator/blob/a8652389ae4de98ca580198594f5a3198080b6de/config/install/manifests.yaml#L2-L13). What am I missing?

Reply:
Authorino CRD belongs to the operator. AuthConfig CRD is the one that belongs to Authorino.

Reply:
Okay I get you now when you said Authorio CRDs you meant Authorino as in Authorino the product and not the CRD kind. Happy that confusion was cleared up. 
So to be sure, are you asking for anything to be changed here?